### PR TITLE
feat(APIM-133): add new env variables to deployment scripts

### PR DESCRIPTION
**Pull request by lmarrai-sw**
## Introduction

APIM-133 had added new environment variables, we need to add them to the deployment scripts, api tests, and docker-compose file.

## Resolution
I've added the new environment variables to the deployment scripts, api tests, and docker-compose file.

### CODE REVIEW

**Review by avaitonis — DISMISSED** [review]
LGTM

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -126,16 +126,18 @@ jobs:
...
-            "APIM_MDM_VALUE=${{ secrets.APIM_MDM_VALUE }}" \
             "GRAPH_BASE_URL=${{ secrets.GRAPH_BASE_URL }}" \
             "GRAPH_AUTHENTICATION_TENANT_ID=${{ secrets.GRAPH_AUTHENTICATION_TENANT_ID }}" \
             "GRAPH_AUTHENTICATION_CLIENT_ID=${{ secrets.GRAPH_AUTHENTICATION_CLIENT_ID }}" \
             "GRAPH_AUTHENTICATION_CLIENT_SECRET=${{ secrets.GRAPH_AUTHENTICATION_CLIENT_SECRET }}" \
             "SHAREPOINT_MAIN_SITE_NAME=${{ secrets.SHAREPOINT_MAIN_SITE_NAME }}" \
             "SHAREPOINT_TFIS_SITE_NAME=${{ secrets.SHAREPOINT_TFIS_SITE_NAME }}" \
-            "SHAREPOINT_TFIS_LIST_ID=${{ secrets.SHAREPOINT_TFIS_LIST_ID }}"
+            "SHAREPOINT_TFIS_LIST_ID=${{ secrets.SHAREPOINT_TFIS_LIST_ID }}" \
+            "MDM_BASE_URL=${{ secrets.MDM_BASE_URL }}" \
```
Comment (abhi-markan) [inline .github/workflows/deployment.yml]: Can we change to `APIM_` since `eStore CA` will be connecting to `MDM CA` via APIM and not directly.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Naming convention update please.

File: .github/workflows/deployment.yml (hunk 1 above)
Comment (lmarrai-sw) [reply .github/workflows/deployment.yml]: I will do, thanks @abhi-markan!

Comment (lmarrai-sw) [reply .github/workflows/deployment.yml]: @abhi-markan I've updated this now

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**